presets/exhibition-preset/src/index.tsx

```diff
@@ -67,11 +67,15 @@ export const exhibitionEditorPreset = extendApp(
     ],
     creators: [
       infoBoxCreator,
-      imageSlideBottomCreator,
-      imageSlideCreator,
-      imageSlideLeftCreator,
-      imageSlideRightCreator,
+      imageServiceSlideCreator, // not showing!!?
+      imageBrowserSlideCreator,
+      // imageSlideBottomCreator,
+      // imageSlideCreator,
+      // imageSlideLeftCreator,
+      // imageSlideRightCreator,
+      emptyCanvasSlide,
       youtubeSlideCreator,
+      imageUrlSlide, // not showing!?
```

Comment:
Two of the creators aren't showing in the list, cant see any console erros  :/

Reply:
Is it the tags? I can see the emptyCanvasSlide has a `Canvas` tag. I can't remember what we are filtering them by